tokio-net/src/process/unix/orphan.rs

```diff
@@ -71,9 +70,9 @@ impl<T: Wait> OrphanQueue<T> for AtomicOrphanQueue<T> {
             match orphan.try_wait() {
                 Ok(Some(_)) => {}
                 Err(e) => error!(
-                    "leaking orphaned process {} due to try_wait() error: {}",
-                    orphan.id(),
-                    e,
+                    message = "leaking orphaned process due to try_wait() error",
+                    orphan.id =orphan.id(),
```

Comment:
This spacing nit made me think: are the macros resilient against whitespace differences?